toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
that is a major challenge that we're going to have to tackle :) With great power comes greater complexity ;)
The whole "we need really, really good tools and really, really good error reporting" comes exactly from the fact that in L20n you *can* do more complex things.

You don't have to, but you can.

@stasm - one thought here is that actually having this as two entities (like before my patch) with one with traits and the other referencing the first one may actually be easier for localizers to understand than select expression.